lib/trento_web/controllers/v1/host_controller.ex

```diff
@@ -200,6 +200,12 @@ defmodule TrentoWeb.V1.HostController do
     end
   end
 
+  def get_policy_resource(%{
+        private: %{phoenix_action: :request_operation},
+        params: %{"operation" => operation}
```

Comment:
could we match on path_params instead of params, so we could match only the params we need and it's more understandable where we are getting the params

https://hexdocs.pm/plug/Plug.Conn.html

Reply:
Alright, let me try that!

Reply:
Yes, as easy as just replacing it. Changed!
Thank you!